### PR TITLE
Devi/video capture sample

### PR DESCRIPTION
**Pull request by v-dmudili**
Created a folder for Video Capture app & added all the project files for windows 10 Iot Enterprise.

Following changes :

1) file modified : VideoCaptureSample.csproj
     Updated the Target Platform Version from 10.0.17763 to 10.0.19041
     Updated the Target Platform MinVersion from 10.0.10240.0 to 10.0.15063

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Generate and Installing App Package procedure.
    Updated the Windows Platform from IoT Core to IoT Enterprise.

### CODE REVIEW

File: VideoCaptureSample/.gitkeep [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline VideoCaptureSample/.gitkeep]: Remove unnecessary file

File: VideoCaptureSample/CS/.gitkeep [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline VideoCaptureSample/CS/.gitkeep]: Remove unnecessary file

File: VideoCaptureSample/CS/Assets/.gitkeep [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline VideoCaptureSample/CS/Assets/.gitkeep]: Remove unnecessary file

File: VideoCaptureSample/CS/Properties/.gitkeep [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline VideoCaptureSample/CS/Properties/.gitkeep]: Remove unnecessary file

File: VideoCaptureSample/CS/README.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+### End Capturing:
+
+* Stops preview and stops capturing video to cameraCapture.wmv file.
+
+### Play Captured Video:
+
+* Reads recorded video data from cameraCapture.wmv file.
+* View stream will appear under Video Review Window section of the canvas.
+
+**NOTE:** On UPBoard , audio output is available via HDMI.
```
Comment (neilsh-msft) [inline VideoCaptureSample/CS/README.md]: remove space between UPBoard and comma  -> "UPBoard ,"

File: VideoCaptureSample/CS/VideoCaptureSample.csproj [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
+    <ProjectGuid>{1A69F528-7B58-480F-B127-6B8F51455D31}</ProjectGuid>
+    <OutputType>AppContainerExe</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>VideoCaptureSample</RootNamespace>
+    <AssemblyName>VideoCaptureSample</AssemblyName>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
```
Comment (neilsh-msft) [inline VideoCaptureSample/CS/VideoCaptureSample.csproj]: Do target platform values need updating from 17763 to 19041?

**Review by rsameser — APPROVED** [review]

File: VideoCaptureSample/.gitkeep (hunk 1 above)
Comment (v-abdrahman) [reply VideoCaptureSample/.gitkeep]: Removed

File: VideoCaptureSample/CS/.gitkeep (hunk 2 above)
Comment (v-abdrahman) [reply VideoCaptureSample/CS/.gitkeep]: Removed

File: VideoCaptureSample/CS/Assets/.gitkeep (hunk 3 above)
Comment (v-abdrahman) [reply VideoCaptureSample/CS/Assets/.gitkeep]: Removed

File: VideoCaptureSample/CS/Properties/.gitkeep (hunk 4 above)
Comment (v-abdrahman) [reply VideoCaptureSample/CS/Properties/.gitkeep]: Removed

File: VideoCaptureSample/CS/VideoCaptureSample.csproj (hunk 6 above)
Comment (v-abdrahman) [reply VideoCaptureSample/CS/VideoCaptureSample.csproj]: Updated the Target Platform Version to 10.0.19041.0

**Pull request merged**